### PR TITLE
MacOS Port 

### PR DESCRIPTION
**Pull request by r614**
MacOS implementation for renv. 
Mentioned in #59

### CODE REVIEW

**Comment by sdhutchins** [comment]
Hey, @r614 ! The issue with Travis CI is with the python versions. 

In the [pyproject.toml](https://github.com/datasnakes/renv/blob/master/pyproject.toml), the python version should be bumped up to `3.5`.

```yaml
[tool.poetry.dependencies]
python = ">=3.4"
click = "^7.0"
pyyaml = "^3.12"
cookiecutter = "^1.6"
```

For the [travis.yml](https://github.com/datasnakes/renv/blob/master/.travis.yml) file, python version 3.4 should be removed.

```yaml
python:
  - 3.4
  - 3.5
  - 3.6
 ```

**Comment by r614** [comment]
Hey @sdhutchins , just fixed it!

**Comment by grabear** [comment]
Ohh!  It's building :)

**Comment by sdhutchins** [comment]
Nice job, @r614 !!!

File: poetry.lock [hunk 1]
```diff
@@ -4,7 +4,7 @@ description = "Better dates and times for Python"
 name = "arrow"
 optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*"
-version = "0.13.1"
+version = "0.13.2"
```
Comment (grabear) [inline poetry.lock]:
```suggestion
version = "0.14.0"
```

**Comment by grabear** [comment]
Bump the project version in the poetry.toml file from `0.3.1` to `0.4.0`

**Review by grabear — CHANGES_REQUESTED** [review]
Bump the project version in the poetry.toml file from `0.3.1` to `0.4.0`

**Comment by r614** [comment]
Updated!

**Review by grabear — APPROVED** [review]

**Comment by grabear** [comment]
Nice job @r614 :)

**Comment by r614** [comment]
Thanks. :D

**Pull request merged**